src/exit/exit.rs

```diff
@@ -0,0 +1,28 @@
+use c_types::c_int;
+use super::_Exit::_Exit;
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn __funcs_on_exit() {}
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn __stdio_exit() {}
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn _fini() {}
+
+#[linkage = "weak"]
+#[no_mangle]
+pub unsafe extern "C" fn __libc_exit_fini() {
+    _fini()
```

Comment:
Based on http://git.musl-libc.org/cgit/musl/tree/src/exit/exit.c#n18 it seems like this should perform some magic-y math on those void pointers before calling _fini(). If that's not necessary for some reason, could you include a comment that explains that here for me and any future readers?